contracts/holder.sol

```diff
@@ -50,12 +50,10 @@ contract Holder is Balanceable, ENSResolvable, Controllable, Transferrable, Toke
     /// @param _ens_ is the address of the ENS registry.
     /// @param _tokenWhitelistNode_ is the ENS node of the Token whitelist.
     /// @param _controllerNode_ is the ENS node of the Controller
-    constructor(address _burnerContract_, address _ens_, bytes32 _tokenWhitelistNode_, bytes32 _controllerNode_)
-        public
-        ENSResolvable(_ens_)
-        Controllable(_controllerNode_)
-        TokenWhitelistable(_tokenWhitelistNode_)
-    {
+    constructor(address _burnerContract_, address _ens_, bytes32 _tokenWhitelistNode_, bytes32 _controllerNode_) public {
```

Comment:
Why did we remove the constructors from all contracts instead of just adding the initialize() interface to the wallet?
I would imagine it still makes sense to invoke regular constructors() for contracts that don't need to be upgradable.

Reply:
+1

Reply:
The `constructor` wasn't removed in independent contracts like holder above, they are not _upgradeable_.

Reply:
noted